docset/windows/nettcpip/set-nettcpsetting.md

```diff
@@ -57,13 +57,16 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
 TCP settings are optimized for different network conditions including latency and congestion.
 To apply a TCP setting to a port number or destination IP address range, create a transport filter by using the New-NetTransportFilter cmdlet.
 
-You can only modify the Custom TCP setting.
+> [!IMPORTANT] 
+>1) You can modify Custom and Non-Custom settings on windows server 2016 and 2019.
+>2) You can modify only Custom settings, Internet and Datacenter settings Cannot be modified on windows 2012 or earlier versions.
```

Comment:
I am not understanding this one. It says you can modify custom settings but not on windows 2012 and earlier? And what about Internet and Data Center settings?

Reply:
> custom

Yes, For 2) You can modify only Custom settings in Windows 2012 or earlier versions. 
So the Internet and datacenter settings are read-only.